<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterPat</class>
 <widget class="QMainWindow" name="ajouterPat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="ajouterPatineur" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>661</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Informations principales</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="TabHockey">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>811</width>
       <height>211</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Hockey</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>321</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Équipe</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Catégorie</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Position</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Latéralité</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Groupe</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Traiteur</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Combiné&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="equipelineEdit">
          <property name="maxLength">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="catlineEdit">
          <property name="maxLength">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="CombinecheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="traiteurCheck">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="Lieulabel">
          <property name="text">
           <string>Lieu</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="LieulineEdit"/>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="gaucheradioButton">
            <property name="text">
             <string>Gaucher</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="droitierradioButton">
            <property name="text">
             <string>Droitier</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="groupeHockcomboBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="posicomboBox">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gardien</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Joueur</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="traiteurWid" native="true">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>261</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="traiteurLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Lundi</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Mardi</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>Mercredi</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Jeudi</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="traitLundilineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="traitMardilineEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="traitMercredilineEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="traitJeudilineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="traiteurTitre">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Traiteur</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Patinage">
      <attribute name="title">
       <string>Patinage Artistique</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>351</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Numéro du club</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Niveau de danse</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Niveau de style libre</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Niveau d'habileté</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Groupe</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="clublineEdit">
          <property name="maxLength">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="danselineEdit">
          <property name="maxLength">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="stylelineEdit">
          <property name="maxLength">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="habiletelineEdit">
          <property name="maxLength">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="groupePAcomboBox"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>PowerSkating</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>281</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Groupe</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>Niveau</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="niveauPowlineEdit">
          <property name="maxLength">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="groupePOWcomboBox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="ValiderpushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>570</y>
       <width>801</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>401</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QLabel" name="IDErrorlabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="adresseErrorlabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="prenomErrorlabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="nomErrorlabel">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nomlineEdit">
        <property name="maxLength">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IDEdit">
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="dateNaissanceErrorLable">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Prénom</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="prenomlineEdit">
        <property name="maxLength">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="courrielErrorLabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Courriel du parent</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QTextEdit" name="condiMedtextEdit"/>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="condiErrorlabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="adresselineEdit">
        <property name="maxLength">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="courriellineEdit">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="maxLength">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="jourlineEdit">
          <property name="maxLength">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_60">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="moislineEdit">
          <property name="maxLength">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_61">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="anneeLineEdit">
          <property name="maxLength">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Date de naissance</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Condition médicale</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>70</y>
       <width>401</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="1">
       <widget class="QLineEdit" name="ParentlineEdit">
        <property name="maxLength">
         <number>35</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="provinceErrorlabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="provincelineEdit">
        <property name="maxLength">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="codePostallineEdit">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="VilleErrorlabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Ville</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="villelineEdit">
        <property name="maxLength">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Code postal</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Province</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Payé</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="cpErrorlabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="parentResErrorlabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Parent responsable</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="payecheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Numéro de téléphone</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QTextEdit" name="NumTeltextEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="numTelErrorlabel">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressources.qrc">:/img/!.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
